aspnetcore/signalr/streaming.md

```diff
@@ -110,33 +160,72 @@ Console.WriteLine("Streaming completed");
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-3.0"
+
+### Client to server streaming
+
+To invoke a client-to-server streaming hub method from the .NET client create a `Channel` and pass the `ChannelReader` as an argument to `SendAsync`, `InvokeAsync`, or `StreamAsChannelAsync`, depending on the hub method being invoked.
```

Comment:
Should we use hyphens for client-to-server and server-to-client in our subsection headers like we do for our prose?

Reply:
Yes (adjective)

Reply:
IMO yes (but OCD)